src/common/globalModel.ts

```diff
@@ -0,0 +1,32 @@
+import { ModelMetadata } from "./ModelMetadata"
+
+enum ModelEnum {
+    user = "user",
+    anotherModel = "anotherModel"
+}
+
+const ModelMetadatas = {
+    [ModelEnum.user]: new ModelMetadata(
+        process.env.USER_DYNAMODB_TABLE || '',
+        {
+            id: "id",
+            name: "name",
+            bob: "bob",
+            address: "address",
+            description: "description",
+            createdAt: "createdAt",
+            updatedAt: "updatedAt",
+            imageUrl: "imageUrl",
+        }),
+    [ModelEnum.anotherModel]: new ModelMetadata(
+        process.env.OTHERMODEL_DYNAMODB_TABLE || '',
+        {
+            id: "id",
+        }),
+}
+
+
+export {
```

Comment:
How do you decide between exporting an object versus export const each object?

Reply:
Well, it depends on the content of the things that I would like to epxort. If they are not string connected one each other, I export the const directly. But in the oposite case, I export them as an object to let importers to handle them in a kind of namespace, meaning that everything inside is related.